### PR TITLE
Add formatRt - see #75

### PR DESCRIPTION
**Pull request by lgatto**

### CODE REVIEW

File: R/formatRt.R [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+##' @return A reformatted retention time.
+##' 
+##' @author Laurent Gatto
+##'
+##' @export
+##'
+##' @examples
+##' formatRt(1524)
+##' formatRt(1)
+##' formatRt("25:24")
```
Comment (jorainer) [inline R/formatRt.R:17]: I was wondering, does this also support `rt` to be of length > 1?

**Review by jorainer — COMMENTED** [review]
Just one comment for now - whether the function is vectorized.

File: R/formatRt.R (hunk 1 above)
Comment (lgatto) [reply R/formatRt.R:17]: It does:
```r
> formatRt(1:10)
 [1] "0:01" "0:02" "0:03" "0:04" "0:05" "0:06" "0:07" "0:08" "0:09" "0:10"
```
I'll mention it in the man page and in the example.

Comment (lgatto) [reply R/formatRt.R:19]: Addressed in latest addition to this PR.

In addition, I also now require an input of length > 0.

**Review by jorainer — APPROVED** [review]
I'm OK with this. Thanks @lgatto

File: R/formatRt.R [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+##' formatRt(1:10)
+##' formatRt("25:24")
+##' formatRt(c("25:24", "25:25", "25:26"))
+formatRt <- function(rt) {
+    stopifnot(length(rt) > 0)
+    ans <- NA
+    if (is.numeric(rt)) {
+        min <- floor(rt / 60)
+        sec <- round(rt - (min * 60))
+        sec <- sprintf("%02d", sec)
```
Comment (sgibb) [inline R/formatRt.R:27]: Why not just:
```r
ans <- sprintf("%d : %02d", min, sec)
```
instead of `sprintf` + `paste`?

File: R/formatRt.R [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        ans <- paste(min, ":", sec, sep = "")        
+    } else if (is.character(rt)) {
+        ans <- strsplit(rt, ":")
+        ans <- sapply(ans, function(x) {
+            x <- as.numeric(x)
+            60 * x[1] + x[2]
+        })
+    } else {
+        warning("Input must be numeric of character.")
```
Comment (sgibb) [inline R/formatRt.R:36]: Shouldn't this be an error?

File: R/formatRt.R [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    }
+    return(ans)
```
Comment (sgibb) [inline R/formatRt.R:38]: `return` isn't needed here.

**Review by sgibb — COMMENTED** [review]
Thanks for the PR @lgatto.

File: R/formatRt.R (hunk 2 above)
Comment (lgatto) [reply R/formatRt.R:27]: Yes, changing it, thanks.

File: R/formatRt.R (hunk 3 above)
Comment (lgatto) [reply R/formatRt.R:36]: To be discussed in #75

**Pull request merged**